python/ray/autoscaler/aws/example-gpu-docker.yaml

```diff
@@ -23,10 +23,17 @@ autoscaling_mode: default
 # and opens all the necessary ports to support the Ray cluster.
 # Empty string means disabled.
 docker:
-    image: "tensorflow/tensorflow:1.12.0-gpu-py3"
+    image: "tensorflow/tensorflow:1.14.0-gpu-py3"
     container_name: "ray-nvidia-docker-test" # e.g. ray_docker
     run_options:
       - --runtime=nvidia
+    mounts:
+        /root/ray_results: /home/ubuntu/ray_results/
+        /root/ray/: /home/ubuntu/ray/
+      # /path/on/container:/path/on/host
```

Comment:
The reason why we want to support this is because in the code, you may depend on `~/ray_results` in your code, but `~` translates to `/root/` on Docker.

But maybe this isn't necessary because you can set your dockerfile...